jmix-flowui/flowui/src/main/java/io/jmix/flowui/download/DownloaderImpl.java

```diff
@@ -108,6 +108,9 @@ public void setFileStorageLocator(FileStorageLocator fileStorageLocator) {
     @Override
     public void setFileStorage(FileStorage fileStorage) {
         this.fileStorage = fileStorage;
+
+        log.warn("From version 2.3 it is no longer necessary to set FileStorage explicitly. " +
```

Comment:
"The passed value is ignored. Actual file storage is obtained from " + FileRef.class.getSimpleName()